src/mock/parsec/mod.rs

```diff
@@ -53,7 +53,9 @@ where
         section_hash: Digest256,
         our_id: S,
         genesis_group: &BTreeSet<S::PublicId>,
+        genesis_related_info: Vec<u8>,
         consensus_mode: ConsensusMode,
+        _secure_rng: Box<dyn rand::Rng>,
```

Comment:
hmm... if it's not used, why added it in?
same to the other place

Reply:
It is used by the real parsec, so the mock has the same interface (except it needs an extra param).